client/components/TestRun/TestRun.css

```diff
@@ -523,3 +523,35 @@ a.btn-options:hover {
 .form-control.is-invalid {
   background-image: none;
 }
+
+.raise-issue-container {
+  display: flex;
+  flex-direction: column;
+  font-size: 0.75rem;
+}
+
+.raise-issue-container .title {
+  align-self: flex-start;
+  font-weight: bold;
+}
+
+.raise-issue-container input[type="radio"] {
+  margin-right: 0.25rem;
+  vertical-align: middle;
+}
+
+.raise-issue-container .options {
+  display: flex;
+}
+
+.raise-issue-container .options.type {
+  justify-content: space-between;
+  margin-bottom: 0.5rem;
+}
+
+.raise-issue-container .options.command {
+  flex-direction: column;
+  align-items: flex-start;
+  text-align: left;
+  margin-bottom: 0.5rem;
```

Comment:
No issue with this. I increased by a somewhat small amount because I felt it shouldn't go _too_ far away from the related 'Raise an Issue' button. But that may also be my preference slipping in as well ha. Any thoughts on an exact value you may prefer here?

Reply:
That works! Thanks for the compromise between our preferences haha.